src/github/github.ts

```diff
@@ -93,17 +101,20 @@ export class Github {
         })
         .on('response', function(response) {
           if (response.statusCode != 200) {
-            let err = new GithubResponseError('Unexpected response from GitHub');
-            err.statusCode = response.statusCode;
-            err.statusMessage = response.statusMessage;
+            let err = new GithubResponseError(
+                response.statusCode,
+                response.statusMessage);
             throw err;
           }
-        })
-        .on('end', () => {
-          resolve();
+          logger.info('Unpacking template files');
```

Comment:
Nit: we've ended all "about to" status updates with an `...`, consider adding here